### ISSUE TITLE
Please make Android tv compatible

### ISSUE THREAD

**Issue by Fxrrcrxdsccexxe** [fast4x/RiMusic#1971]
### Description

Please make this app for Android TV fully compatible because in this some feature r not working properly

### Suggested Solution

In Android TV very difficult to navigate so plz 🙏 fix some bugs and give fully support for Android TV

### Alternatives

... 

### Additional Context

...

**Comment by fast4x** [comment]
Yes it's planned, I'll see after new layout and functionality development.

**Comment by Fxrrcrxdsccexxe** [comment]
Hello, 
How to back up inner tune data in Ri music

**Comment by fast4x** [comment]
Hello I have a project that will allow you that still is not ready.
